### PR TITLE
Add error handling to #loadJs

### PR DESCRIPTION
**Pull request by ejoubaud**
Hi,

Thanks a lot for `meteor-external-file-loader`, very useful.

I needed a way to handle error cases (e.g. display an error/retry message to the user if I can't process because they got disconnected or something went wrong with the lib's loading)

Turns out `jQuery.getScript` already returns a promise, so it's pretty straightforward to allow error handling to your lib's user with something like that:

``` js
Loader.prototype.loadJs('url', successCallback, 10000).fail(errorCallback)
```

Thought it might be useful to other people too.

### CODE REVIEW

File: lib/external-file-loader.js [hunk 1]
```diff
@@ -7,8 +7,16 @@
 	}
 
 	// Callback must be of the jquery success callback form: function(data, textStatus, jqxhr)
-	Loader.prototype.loadJs = function(url, callback) {
-		var self = this;
+	Loader.prototype.loadJs = function(url, callback, timeoutDelay) {
```
Comment (davidd8) [inline lib/external-file-loader.js:12]: Can you add a description for the timeoutDelay function parameter?

**Comment by davidd8** [comment]
Thanks for the addition! Can you add an example timeout use case to the readme, and address my comment, and I'll be happy to merge.

**Comment by ejoubaud** [comment]
Done, let me know if that's good enough :)

**Comment by davidd8** [comment]
Great, thanks!

**Pull request merged**